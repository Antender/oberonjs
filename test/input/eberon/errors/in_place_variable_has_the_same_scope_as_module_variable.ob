MODULE m1;
VAR i: INTEGER;
BEGIN
    FOR i <- 1 TO 10 END;
END m1.