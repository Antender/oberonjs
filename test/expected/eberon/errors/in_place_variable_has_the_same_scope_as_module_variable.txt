line 4: 'i' already declared in module scope
